vpr/src/place/simpleRL_move_generator.h

```diff
@@ -0,0 +1,81 @@
+#ifndef VPR_SIMPLERL_MOVE_GEN_H
+#define VPR_SIMPLERL_MOVE_GEN_H
+#include "move_generator.h"
+#include "median_move_generator.h"
+#include "weighted_median_move_generator.h"
+#include "feasible_region_move_generator.h"
+#include "weighted_centroid_move_generator.h"
+#include "uniform_move_generator.h"
+#include "critical_uniform_move_generator.h"
+#include "centroid_move_generator.h"
+
+class KArmedBanditAgent {
+  public:
+    virtual ~KArmedBanditAgent() {}
+    virtual void set_k(size_t k) = 0;
```

Comment:
set_k doesn't seem like a great name for this function; it really is more of a constructor.  Could rename it initialize (int k), or move the current set_k code into the constructors of the derived classes (and have them take a k parameter) since it really looks like constructor code.

Reply:
Fixed by removing this function and moving its code to the constructor of each class